source/administration/config-settings.rst

```diff
@@ -2093,6 +2093,15 @@ See the `Guest Accounts documentation <https://docs.mattermost.com/deployment/gu
 | This feature's ``config.json`` setting is ``"GuestFilter": ""`` with string input.                                                                                   |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Admin Filter
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+*Available in Enterprise Edition E20*
+
+(Optional) Enter a filter to use for designating the System Admin role to users. The user is promoted to this role on their next login or at the next scheduled AD/LDAP sync.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
Config.setting formatting not rendering.

Reply:
Fixed.